CMake/External_OpenCV.cmake

```diff
@@ -21,6 +24,23 @@ else()
   set(fletch_ENABLE_OpenCV_FFmpeg CACHE INTERNAL FALSE)
 endif()
 
+# Allow OpenCV's VTK option to be explicitly turned off
+if(fletch_ENABLE_VTK)
+  # By default turn opencv-VTK support off, even when building with VTK
+  option(fletch_ENABLE_OpenCV_VTK "Build OpenCV with VTK support" FALSE )
+  mark_as_advanced(fletch_ENABLE_OpenCV_VTK)
+else()
+  set(fletch_ENABLE_OpenCV_VTK CACHE INTERNAL FALSE)
```

Comment:
yeah, but once you set a variable to INTERNAL, it never shows up again in the GUI
So if you go back and try to turn it back on via the GUI, you can't...

Yeah, CUDA and FFmpeg need to be updated on this PR or mine, yours if pushing forward, so I vote you do it here. I can cherry pick another PR with Dave when he gets to mine if need be.